DockerfileWithHubble

```diff
@@ -0,0 +1,51 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM openjdk:11-slim
+
+LABEL maintainer="HugeGraph Docker Maintainers <dev@hugegraph.apache.org>"
+
+ENV JAVA_OPTS="-XX:+UnlockExperimentalVMOptions -XX:+UseContainerSupport -XX:MaxRAMPercentage=50 -XshowSettings:vm"
+
+COPY . /hugegraph/hugegraph-server
+WORKDIR /hugegraph/hugegraph-server
+
+# 1. Install environment
+RUN set -x \
+    && apt-get -q update \
+    && apt-get -q install -y --no-install-recommends --no-install-suggests \
+       dumb-init \
+       procps \
+       curl \
+       lsof \
+    && apt-get clean
+
+# 2. Init HugeGraph Sever
+RUN set -e \
+    && cd /hugegraph/hugegraph-server/ \
+    && sed -i "s/^restserver.url.*$/restserver.url=http:\/\/0.0.0.0:8080/g" ./conf/rest-server.properties \
+    && ./bin/init-store.sh
+
+# 3. Init HugeGraph Hubble
+RUN set -e \
+    && cd /hugegraph/hugegraph-server/hugegraph-hubble \
```

Comment:
where is the hubble? it's in toolchain repo now

Reply:
see: https://github.com/apache/incubator-hugegraph/pull/2046/files#diff-7987535fc6eb90c084a6031a104dc578fc6c9d282fb18d8ce5b034f41b737f76R32-R34

Reply:
> see: [#2046 (files)](https://github.com/apache/incubator-hugegraph/pull/2046/files#diff-7987535fc6eb90c084a6031a104dc578fc6c9d282fb18d8ce5b034f41b737f76R32-R34)

who call it? (or just leave for future hubble download URI?)

Reply:
> > see: [#2046 (files)](https://github.com/apache/incubator-hugegraph/pull/2046/files#diff-7987535fc6eb90c084a6031a104dc578fc6c9d282fb18d8ce5b034f41b737f76R32-R34)
> 
> who call it? (or just leave for future hubble download URI?)

just leave for future hubble download URI